e2e/test/scenarios/question/notebook-native-preview-sidebar.cy.spec.ts

```diff
@@ -19,8 +19,7 @@ describe("scenarios > question > notebook > native query preview sidebar", () =>
     );
   });
 
-  it("should not an show empty sidebar when no data source is selected", () => {
-    H.activateToken("pro-self-hosted");
+  it("should not show empty sidebar when no data source is selected", () => {
```

Comment:
Probably makes more sense for this to be a unit test.
Outside the scope of this PR, though.